### DISCUSSION TITLE
Where is ENABLE_QUAD_PRECISION supported?

### DISCUSSION THREAD

**Discussion by climbfuji** [NOAA-GFDL/FMS#875] [General]
I came across a weird problem on my newest MacBook Pro, which has Apple's latest M1 chip. With -DENABLE_QUAD_PRECISION=ON, I got the following error when compiling (through spack):
```
Error: Kind -1 not supported for type REAL at (1)
```
This is with Apple's clang-13.0.0 and gfortran-11.2.0. I am almost certain that I could compile with quad precision enabled with the OS and same compiler versions on my other MacBook Air, which has an Intel i5 chip.

Does anyone know if there is such a limitation for the M1 chip? Or is it a problem with my system. Thanks ...

**Comment by climbfuji** [comment]
I am almost certain that this is an Apple Silicon thing. I ran into a similar problem with the stochastic physics packagem see here: https://github.com/noaa-psd/stochastic_physics/issues/52

**Comment by underwoo** [comment]
It is likely a mac silicon issue.  I'll have to run a few tests on my M1 mac.  I still mainly use my Intel Mac for development.  

Autoconf has a test to see if quad precision is supported.  The CMake build needs to be updated to run some of the same tests.
